website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -81,20 +129,14 @@ Disclaimer: The bootstrap CLI will ONLY use the private key to push WGE resource
 ### Selecting WGE version
 The bootstrap CLI will prompt you to choose from the latest 3 versions of Weave GitOps Enterprise.
 
-### Create admin credentials
+### Create Cluster User
 You will be prompt to provide admin username and password, which will be used to access the dashboard. This will create admin secret with the credentials. If you already have previous admin credentials on your cluster, the installation will prompt you if you want to continue with the old credentials or exit and revoke them and re-run the installation.
 
 ### Select the domain
-To access Weave GitOps Enterprise dashboard, you have one of 2 options: 
-##### via localhost
-This requires no pre-requisites and more suitable for testing on your local or development environment, or air-gapped environments.
+To access Weave GitOps Enterprise dashboard, you have one of 2 options:
 
-
-##### via External Domain
-This is more suitable for production environments where multiple users are going to access the dashboard. However, this will require extrnal dns to be set up and configured for the domain to be accessible. 
-:::tip
-While there are multiple ways to setup and configure external dns. You can find a full example on how to setup it for multiple cloud providers [here](#external-dns)
-:::
+1. **Localhost**: requires no pre-requisites and more suitable for testing on your local or development environment, or air-gapped environments.
```

Comment:
rephrase this paragraph with the explicit dependencies, and to follow up in the existing ticket